drop-down-common.ts

```diff
@@ -14,20 +14,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ***************************************************************************** */
 import { ObservableArray } from "data/observable-array";
-import { CSSType, CoercibleProperty, EventData, Property, View } from "ui/core/view";
+import { AddChildFromBuilder, CSSType, CoercibleProperty, EventData, Property, View } from "ui/core/view";
 import { addWeakEventListener, removeWeakEventListener } from "ui/core/weak-event-listener";
 import { ItemsSource } from "ui/list-picker";
+import { Style } from "ui/styling/style";
+import { TextBase } from "ui/text-base";
 import * as types from "utils/types";
 import { DropDown as DropDownDefinition, SelectedIndexChangedEventData, ValueItem, ValueList as ValueListDefinition } from ".";
 
 export * from "ui/core/view";
 
+@CSSType("DropDownList")
+export class DropDownList extends TextBase {
+    
+}
+
 @CSSType("DropDown")
-export abstract class DropDownBase extends View implements DropDownDefinition {
+export abstract class DropDownBase extends View implements DropDownDefinition, AddChildFromBuilder {
     public static openedEvent = "opened";
     public static closedEvent = "closed";
     public static selectedIndexChangedEvent = "selectedIndexChanged";
 
+    public dropDownList: DropDownList;
```

Comment:
I think this can be safely made `private`
```suggestion
    private dropDownList: DropDownList;
```